VaticanBAV/et/BAVet117.xml

```diff
@@ -26,29 +26,152 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <repository ref="INS0003BAV"/>
                         <idno>Aeth. 117</idno>
                     </msIdentifier>
-                    <msContents>
-                      <msItem xml:id="ms_i1">
+
```

Comment:
for composite MSS we can still have an empty msContents with an empty summary, then the summary of the whole will be printed in the app

```
<msContents>
<summary/>
</msContents>
```